livestream-compose-sample/src/main/java/io/getstream/livestream/compose/players/YoutubePlayer.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.livestream.compose.players
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.YouTubePlayer
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.listeners.AbstractYouTubePlayerListener
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.options.IFramePlayerOptions
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView
+
+/**
+ * View component which adds YoutubePlayer to play youtube videos
+ *
+ * @param modifier - Modifier for styling.
+ */
+@Composable
+fun YoutubePlayer(modifier: Modifier = Modifier) {
+    AndroidView(modifier = modifier.fillMaxSize(), factory = { context ->
+        val playerListener = object : AbstractYouTubePlayerListener() {
+            override fun onReady(youTubePlayer: YouTubePlayer) {
+                youTubePlayer.loadVideo(videoId = "XYqrrpvTtU8", startSeconds = 0f)
```

Comment:
We should add the `videoId` to the parameters. Maybe even allow for the `playerOptions` to be passed in.

Reply:
Good point